docs/Displaying a User Interface/Template Images/index.md

```diff
@@ -94,6 +94,43 @@ sdlManager.getScreenManager().setPrimaryGraphic(null);
 ```
 !@
 
+### Overwriting Images
+When a file is to be uploaded to the module, the @![iOS]`SDLFileManager`!@@![android, javaSE, javaEE, javascript]`FileManager`!@ checks if a file with the same name has already been uploaded to module and skips the upload if the image has already been uploaded. For cases where an image by the same name needs to be reuploaded then the @![iOS]`SDLArtwork`'s !@@![android, javaSE, javaEE, javascript]`SdlArtwork`'s!@ `overwrite` property should be used. Setting `overwrite` to `true` before passing the image to a @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE, javascript]`ScreenManager`!@ method such as @![iOS]`primaryGraphic` and `secondaryGraphic`!@@![android, javaSE, javaEE, javascript]`setPrimaryGraphic()` and `setSecondaryGraphic()`!@ will force the image to be reuploaded. This includes methods such as @![iOS]`preloadChoices:withCompletionHandler:`!@@![android, javaSE, javaEE, javascript]`preloadChoices()`!@ where the arguments passed in contain images.
+
+!!! IMPORTANT
+Please note that many production modules on the road do not refresh the HMI with the new image if the file name has not changed. If you want the image to refresh on the screen immediately, we suggest using two image names and toggling back and forth between the names each time you update the image. 
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+SDLArtwork *sdlArtwork = <#SDLArtwork#>;
+sdlArtwork.overwrite = YES;
+sdlManager.screenManager.primaryGraphic = sdlArtwork;
+```
+
+##### Swift
+```swift
+let sdlArtwork = <#SDLArtwork#>
+sdlArtwork.overwrite = true
+sdlManager.screenManager.primaryGraphic = sdlArtwork
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+// TODO: Code that sets an artwork's overwrite property to true before passing to setPrimaryGraphic()
+```
+!@
+
+@![javascript]
+```js
+// setting the primary graphic with overwrite set to true
+sdlArtwork.setOverwrite(true);
+sdlManager.getScreenManager().setPrimaryGraphic(sdlArtwork);
+```
+!@
```

Comment:
```suggestion
```

Because this is all not recommended, I suggest removing code showing how to do it.